### PR TITLE
Remove mixermon option (make it always true)

### LINKED ISSUE #4557: Remove MIXERS_MONITOR option, make MM always available.
Hy,
Profil X9D+

No Mixermonitor at Simulator with ENTER
At Option Mixermonitor is selceted

Same with  X9E

### PR DESCRIPTION
**Pull request by 3djc**
This fixes #4557

### CODE REVIEW

**Comment by mpaperno** [comment]
👍  

I hope you get as much satisfaction from removing code as I do.  :)

**Comment by kilrah** [comment]
Maybe FW build script too?

**Comment by bsongis** [comment]
@mpaperno there are potential satisfaction for a couple of months in opentx :)

**Comment by lthole** [comment]
Does this mean that the mixermon screen will be implemented on the X7?

**Comment by schwabe** [comment]
@lthole unrelated to that, this commit only affects radios that already have a mixermon.

File: radio/src/gui/128x64/view_channels.cpp [hunk 1]
```diff
@@ -39,23 +38,20 @@ void menuChannelsView(event_t event)
...
   else
     ch = 0;
 
-#ifdef MIXERS_MONITOR
   if (mixersView)
   lcdDrawTextAlignedCenter(0*FH, MIXERS_MONITOR);
   else
-#endif
+
   lcdDrawTextAlignedCenter(0*FH, CHANNELS_MONITOR);
```
Comment (mpaperno) [inline radio/src/gui/128x64/view_channels.cpp]: These could use some indents I think.  Also you'd get another -1 LoC (if you don't use braces).  :-)
Or even `lcdDrawTextAlignedCenter(0, mixersView ? MIXERS_MONITOR : CHANNELS_MONITOR);`  🤓

Comment (mpaperno) [inline radio/src/gui/128x64/view_channels.cpp]: Sorry, I didn't realize `lcdDrawTextAlignedCenter` was a macro.  If you put the conditional in parenthesis, it works (tested on AVR compiler).  I guess that's a good reason to always use brackets in such cases (note to self).  

`lcdDrawTextAlignedCenter(0, (mixersView ? MIXERS_MONITOR : CHANNELS_MONITOR));`

(Not saying you need to change this, but just FYI).

Comment (mpaperno) [inline radio/src/gui/128x64/view_channels.cpp]: Never mind, sorry.... still doesn't work as one line.  Tested with wrong compiler!

**Comment by bsongis** [comment]
👍

**Pull request merged**